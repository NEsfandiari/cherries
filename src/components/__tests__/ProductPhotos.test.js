import React from 'react'
import renderer from 'react-test-renderer'
import ProductPhotos from '../molecules/ProductPhotos'

describe('ProductPhotos', () => {
  it('renders correctly', () => {
    const images = [
      {
        file: {
          url:
            '//images.ctfassets.net/4yejk6rho2rp/6pSrs6IctGUKiQamkkqEaQ/3f917708b9e0ccd1a33802c8e6be3ee6/Temperament-female-contracted-joker-metal-fashionable-abstract-gestures-heart-first-heart-students-joker-earrings-stud-earri.jpg',
        },
      },
      {
        file: {
          url:
            '//images.ctfassets.net/4yejk6rho2rp/3Ug6cvNybmquCqam8wKsWs/1563854d7d1673e0e943af7c3018d9d2/Temperament-female-contracted-joker-metal-fashionable-abstract-gestures-heart-first-heart-students-joker-earrings-stud-earri.jpg',
        },
      },
      {
        file: {
          url:
            '//images.ctfassets.net/4yejk6rho2rp/3r9KcbXdu0ssUaMGMqcqI0/7567b629b759930d54d2e714a995aa27/Temperament-female-contracted-joker-metal-fashionable-abstract-gestures-heart-first-heart-students-joker-earrings-stud-earri.jpg',
        },
      },
    ]
    const tree = renderer.create(<ProductPhotos images={images} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
