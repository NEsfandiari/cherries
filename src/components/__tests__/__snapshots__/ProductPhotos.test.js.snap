// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductPhotos renders correctly 1`] = `
<div
  className="sc-bdVaJa bYrJrv"
>
  <img
    alt="Highlighted Product"
    className="highlight animated fadeIn"
    src={
      Object {
        "file": Object {
          "url": "//images.ctfassets.net/4yejk6rho2rp/6pSrs6IctGUKiQamkkqEaQ/3f917708b9e0ccd1a33802c8e6be3ee6/Temperament-female-contracted-joker-metal-fashionable-abstract-gestures-heart-first-heart-students-joker-earrings-stud-earri.jpg",
        },
      }
    }
  />
  <div
    className="photoCarousel animated slideInRight"
  >
    <img
      alt="Product Page Image0"
      name={0}
      onClick={[Function]}
      src={
        Object {
          "file": Object {
            "url": "//images.ctfassets.net/4yejk6rho2rp/6pSrs6IctGUKiQamkkqEaQ/3f917708b9e0ccd1a33802c8e6be3ee6/Temperament-female-contracted-joker-metal-fashionable-abstract-gestures-heart-first-heart-students-joker-earrings-stud-earri.jpg",
          },
        }
      }
      style={
        Object {
          "opacity": 1,
        }
      }
    />
    <img
      alt="Product Page Image1"
      name={1}
      onClick={[Function]}
      src={
        Object {
          "file": Object {
            "url": "//images.ctfassets.net/4yejk6rho2rp/3Ug6cvNybmquCqam8wKsWs/1563854d7d1673e0e943af7c3018d9d2/Temperament-female-contracted-joker-metal-fashionable-abstract-gestures-heart-first-heart-students-joker-earrings-stud-earri.jpg",
          },
        }
      }
      style={Object {}}
    />
    <img
      alt="Product Page Image2"
      name={2}
      onClick={[Function]}
      src={
        Object {
          "file": Object {
            "url": "//images.ctfassets.net/4yejk6rho2rp/3r9KcbXdu0ssUaMGMqcqI0/7567b629b759930d54d2e714a995aa27/Temperament-female-contracted-joker-metal-fashionable-abstract-gestures-heart-first-heart-students-joker-earrings-stud-earri.jpg",
          },
        }
      }
      style={Object {}}
    />
  </div>
</div>
`;
